libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryClient.java

```diff
@@ -24,16 +24,19 @@
   private static final String EXTRA_DEBUGGING_LOG = "Sending POST to %s with %d event(s) (user agent: %s) "
     + "with payload: %s";
 
-  private String accessToken = null;
-  private String userAgent = null;
+  private String accessToken;
```

Comment:
I like how these member variables no longer have redundant initializations but could we also make them immutable?

Reply:
Both can be updated, so making them `final` would require removing that functionality.